AudioBar/AudioBarViewController.swift

```diff
@@ -82,12 +90,16 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
                     beginObservingPlayerItem(playerItem)
                     player.replaceCurrentItem(with: playerItem)
                 }
+            case .start(withDuration: let duration):
+                player.play()
+                nowPlayingInfoCenter.nowPlayingInfo![MPMediaItemPropertyPlaybackDuration] = duration
             case .play:
                 player.play()
             case .pause:
                 player.pause()
             case .setCurrentTime(let time):
                 player.seek(to: CMTime(timeInterval: time))
+                nowPlayingInfoCenter.nowPlayingInfo![MPNowPlayingInfoPropertyElapsedPlaybackTime] = time
```

Comment:
Why isn't this done declaratively in `didUpdate view`? Also, `.player` messages are for the player. It's in the name! Player only.